python_modules/libraries/dagster-dg-cli/dagster_dg_cli/cli/api/formatters.py

```diff
@@ -61,6 +62,15 @@ def _format_timestamp(timestamp: float) -> str:
         return f"Invalid timestamp: {timestamp}"
 
 
+def _format_agent_timestamp(timestamp: float) -> str:
```

Comment:
Does this need to be different from the existing timestamp formatter? Seems like we should use a common utility in this layer